frontend/src/app/main/ui/shapes/path.cljs

```diff
@@ -7,28 +7,35 @@
 (ns app.main.ui.shapes.path
   (:require
    [app.common.logging :as log]
+   [app.common.types.path :as path]
    [app.main.ui.shapes.custom-stroke :refer [shape-custom-strokes]]
-   [app.util.object :as obj]
-   [app.util.path.format :as upf]
    [rumext.v2 :as mf]))
 
+(defn- content->string
```

Comment:
Is this the better place for this function? don't we need it elsewhere?

Reply:
This function is specific on this namespace and only used here, because the normal process is just doing the `.toString`. This function add some safety checks for the render of the component purpose.